api/src/main/java/au/org/aodn/nrmn/restapi/validation/validators/global/formatted/DuplicateRowCheck.java

```diff
@@ -36,10 +37,9 @@ public DuplicateRowCheck() {
                 .map(duplicateRow ->
                         invalid(
                                 job.getId(),
-                                String.format("Row %s may be a duplicate of row %s.",
-                                        duplicateRow.getRef().getPos(),
-                                        duplicateRow.getRef().getPos() - 1),
-                                ValidationLevel.WARNING)
+                                "Row may be a duplicate of preceding row",
```

Comment:
Remove 'of preceding row' stil duplicates should be detected even if they are not grouped together.

Reply:
The logic only finds duplicates that immediately follow the original row